Automattic/wordpress-activitypub
# Editor: avoid errors when switching to edit a Pattern block

We get this error when transitioning to editing a synced block.

```
The "activitypub-editor-plugin" plugin has encountered an error and cannot be rendered.
```

The console error:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'activitypub_content_visibility')
```

## Proposed changes:
* exit early when `postType === wp_block`
* error check 

### Other information:


## Testing instructions:
1. On a post, convert any block to a reusable Pattern block (click three dots, "Create Pattern," ensure "Synced" is on)
2. Click "Edit original" on the synced pattern
3. No error with this PR, but `The "activitypub-editor-plugin" plugin has encountered an error and cannot be rendered` without it.
